### DISCUSSION TITLE
获取设置失败： 请稍候，正在加载储存

### DISCUSSION THREAD

**Discussion by 578066229** [AlistGo/alist#5070] [General]
获取设置失败： 请稍候，正在加载储存

有人遇到吗，怎么处理的

**Comment by falwiw** [comment]
同样的问题

**Comment by sliasyang** [comment]
阿里云盘open接口AIP 挂了 等一般时间超时过去了进去改掉

**Reply by falwiw** [reply]
感谢回复，我以为是程序挂了，重新安装了Alist,但阿里的token获取不到了
![image](https://github.com/alist-org/alist/assets/14345904/69087015-1ffc-44f0-9f07-d7e7ee72cbcc)

**Reply by sliasyang** [reply]
我也发现了，好像三个API都访问不到了。。。等大佬提供新的API地址把

**Reply by falwiw** [reply]
好的 感谢！！！

**Reply by sliasyang** [reply]
https://api.xhofe.top/alist/ali_open/token
这个API已经恢复正常

**Comment by Troray** [comment]
> 现在什么令牌都更新不到。完全废了。

这边建议你卸载Alist。
